backend/main/models/proposal.go

```diff
@@ -200,11 +199,20 @@ func (p *Proposal) UpdateDraftProposal(db *s.Database) error {
 		end_time = COALESCE($7, end_time),
 		body = COALESCE($8, body),
 		block_height = COALESCE($9, block_height),
-		cid = COALESCE($10, cid),
-		composite_signatures = COALESCE($11, composite_signatures),
-		voucher = COALESCE($12, voucher)
-		WHERE id = $13
-	`, p.Name, p.Choices, p.Strategy, p.Min_balance, p.Max_weight, p.Start_time, p.End_time, p.Body, p.Block_height, p.Cid, p.Composite_signatures, p.Voucher, p.ID)
```

Comment:
maybe this is another ticket, but doesnt look like there is currently a way to take a proposal from "draft" to "published"... should `status` be included as part of `UpdateDraftProposal`, or should it be a separate endpoint/action?

Reply:
oh nevermind, i addressed this elsewhere in another comment!